src/main/groovy/wooga/gradle/appcenter/AppCenterConventions.groovy

```diff
@@ -0,0 +1,37 @@
+/*
+ * Copyright 2019-2021 Wooga GmbH
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *       http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ */
+
+package wooga.gradle.appcenter
+
+import com.wooga.gradle.PropertyLookup
+
+class AppCenterConventions {
```

Comment:
technically you can't simply rename the class as the original class `AppCenterConsts` is part of the public API and would mean a breaking API change.

Reply:
I see.